prow/report/report.go

```diff
@@ -109,22 +113,32 @@ func Report(ghc GithubClient, reportTemplate *template.Template, pj v1.ProwJob)
 	if ghc == nil {
 		return fmt.Errorf("trying to report pj %s, but found empty github client", pj.ObjectMeta.Name)
 	}
+
 	if !pj.Spec.Report {
 		return nil
 	}
+
 	refs := pj.Spec.Refs
-	if len(refs.Pulls) != 1 {
-		return fmt.Errorf("prowjob %s has %d pulls, not 1", pj.ObjectMeta.Name, len(refs.Pulls))
+	if len(refs.Pulls) > 1 {
+		// we are not reporting for batch jobs
```

Comment:
nit: I'd generally put this at the end of the if line

Reply:
done